core/tauri-runtime-wry/src/lib.rs

```diff
@@ -950,6 +951,12 @@ impl WindowBuilder for WindowBuilderWrapper {
   fn get_menu(&self) -> Option<&Menu> {
     self.menu.as_ref()
   }
+
+  #[cfg(target_os = "linux")]
+  fn disable_mouse_event(mut self, disable: bool) -> Self {
```

Comment:
After thinking about it, I think there's no reason to expose a method for it. We should just disable it in `with_config`.

Reply:
Thanks for your comment! I made some changes according to it. Is my understanding correct? If not, please give me some suggestion of the code

https://github.com/tauri-apps/tauri/pull/6025/commits/5d1e22a28cb011771f8cac9eac034549e3169400

Reply:
Sorry this is not what I mean. I should comment in `config.rs` instead.
What you should do is:
- Remove `disable_mouse_event` in config.rs
- Bring this method back
- Add this trait method in `core/tauri-runtime/src/webview.rs`

And the code block you added should work.
Please at least run the build once on your device.
Also you need to add a change file to explain the change.